docs/playground/src/PlaygroundConfiguration.ts

```diff
@@ -304,18 +304,18 @@ export const PlaygroundConfiguration: IStringMap<IComponentPlaygroundConfigurati
               className: 'result-template',
               type: 'text/underscore'
             },
-            new SectionBuilder()
-              .withComponent(
```

Comment:
Hmm, why is `.withComponent` a problem here when it is used in the rest of the file? Is it because it is contained inside a script tag?